indico/modules/categories/client/js/components/CategoryStatistics.jsx

```diff
@@ -0,0 +1,193 @@
+                  axes={axes}
+                  tooltip={tooltip}
+                />
+              </div>
+            )}
+          </Segment>
+        </Segment>
+        <Segment>
+          <Header as="h2">
+            <Header.Content>
+              <Translate>Number of contributions</Translate>
+              <Header.Subheader>
+                <Translate>The year is the one of the start date of the contribution.</Translate>
+              </Header.Subheader>
+            </Header.Content>
+          </Header>
+          <Segment basic>
+            {contributionsGraph[0].data.length === 0 ? (
+              <Message info>
+                <Translate>No data available</Translate>
+              </Message>
+            ) : (
+              <div className="chart-container">
+                <Chart
+                  className="chart"
+                  data={contributionsGraph}
+                  series={series}
+                  axes={axes}
+                  tooltip={tooltip}
+                />
+              </div>
+            )}
+          </Segment>
+        </Segment>
+      </Segment.Group>
+      <Segment.Group horizontal>
+        <Segment inverted color="blue" textAlign="center">
+          <Statistic inverted>
+            <Statistic.Value>{totalEvents}</Statistic.Value>
+            <Statistic.Label>
+              <Translate>Total number of events</Translate>
+            </Statistic.Label>
+          </Statistic>
+        </Segment>
+        <Segment inverted color="blue" textAlign="center">
+          <Statistic inverted>
+            <Statistic.Value>{totalContributions}</Statistic.Value>
+            <Statistic.Label>
+              <Translate>Total number of contributions</Translate>
+            </Statistic.Label>
+          </Statistic>
+        </Segment>
+        <Segment inverted color="blue" textAlign="center">
+          <Statistic inverted>
+            <Statistic.Value>{files}</Statistic.Value>
+            <Statistic.Label>
+              <Translate>Number of attachments</Translate>
+            </Statistic.Label>
+          </Statistic>
+        </Segment>
+        {data.users && (
+          <Segment inverted color="blue" textAlign="center">
+            <Statistic inverted>
+              <Statistic.Value>{users}</Statistic.Value>
+              <Statistic.Label>
+                <Translate>Number of users</Translate>
+              </Statistic.Label>
+            </Statistic>
+          </Segment>
+        )}
+      </Segment.Group>
+      <Translate>Last updated on:</Translate> {updatedTime}
+    </Container>
+  );
+}
+
+CategoryStatisticsDisplay.propTypes = {
+  data: PropTypes.shape({
+    contributions: PropTypes.object,
+    events: PropTypes.object,
+    files: PropTypes.number,
+    updated: PropTypes.string,
+    users: PropTypes.number,
+    totalEvents: PropTypes.number,
+    totalContributions: PropTypes.number,
+    minYear: PropTypes.number,
+    maxYear: PropTypes.number,
```

Comment:
I think all of them could be `isRequired`

Reply:
Apart from `users`.